.github/workflows/release.yml

```diff
@@ -62,12 +69,14 @@ jobs:
     - name: Install fpm
       uses: fortran-lang/setup-fpm@v3
       with:
-        fpm-version: 'v0.4.0'
+        fpm-version: 'v0.8.0'
 
     - name: Create single file version
       run: |
         echo "OUTPUT=${{ env.OUTPUT }}" >> $GITHUB_ENV
         echo "#define FPM_BOOTSTRAP" > fpm-${{ env.VERSION }}.F90
+        # We need to pass the exact version string that a fpm build command would send
+        echo "#define FPM_RELEASE_VERSION ${{ env.MANIFEST_VERSION }}" > fpm-${{ env.VERSION }}.F90
```

Comment:
2) manually set a preprocessor flag. 

This is OK I think, but now after merging with fpm 0.8.0 the CI is broken again: I see there are now C files which are not included in the single-file-source, which seems to break this action

Reply:
@awvwgk ideas how to solve this latest one? I'm guessing it won't be trivial (create a 2-file version, *.f* and *.c?)